librz/util/table.c

```diff
@@ -60,16 +61,14 @@ static void __table_adjust(RzTable *t) {
 	}
 }
 
-RZ_API void rz_table_row_free(void *_row) {
+RZ_API void rz_table_row_free(void *_row, void *user) {
 	RzTableRow *row = _row;
-	rz_list_free(row->items);
-	free(row);
+	rz_pvector_free(row->items);
 }
 
-RZ_API void rz_table_column_free(void *_col) {
+RZ_API void rz_table_column_free(void *_col, void *user) {
```

Comment:
what is the purpose of the new `user` pointer?

Reply:
Please handle NULL case.

Reply:
create a static function with the right signature and call rz_table_*_free from there. the API should remain clean and independent of the storage we are using inside the implementation.